packages/expo-cli/src/commands/eas-build/build/builders/AndroidBuilder.ts

```diff
@@ -183,9 +182,7 @@ class AndroidBuilder implements Builder<Platform.Android> {
   ): Promise<Partial<Android.ManagedJob>> {
     return {
       ...(await this.prepareJobCommonAsync(archiveUrl)),
-      type: BuildType.Managed,
-      packageJson: { example: 'packageJson' },
```

Comment:
`packageJson` and `manifest` weren't included in the `ManagedJob` type and so i removed them